### ISSUE TITLE
[Bug Report] App showing JS error while opening

### ISSUE THREAD

**Issue by Gustav0ar** [cstayyab/WALC#39]
**Describe the bug**
App shows error when opening

**WALC Installation Type:**  Snap

**To Reproduce**
1. Open the app

**Expected behavior**
Not show error

**Did you try closing and then opening it back?:** Yes

**Screenshots**

![walc](https://user-images.githubusercontent.com/628398/82452325-7b689200-9a85-11ea-8769-0d3523d3ea2a.png)

**System Information (please complete the following information):**
 - OS/Distro: PopOS 20.04

**Additional information.**
App works fine even with the error

**Comment by Zzombiee2361** [comment]
Huh, that's weird. Maybe `lsb_release` doesn't work on snap due to sandbox? I'm not familiar with snaps but try giving it more permission

**Comment by cstayyab** [comment]
> Maybe lsb_release doesn't work on snap due to sandbox?

`lsb_release` works with snaps I have checked that. 

@Gustav0ar  can you please provide the output of the following command from your Terminal:
```
lsb_release -a
```

**Comment by Gustav0ar** [comment]
@cstayyab 

No LSB modules are available.
Distributor ID: Pop
Description:    Pop!_OS 20.04 LTS
Release:        20.04
Codename:       focal

**Comment by cstayyab** [comment]
According to the output you have provided this error should not occur. I will have to look into it in more details.

**Comment by erduende** [comment]
When I start WALC it shows the same error window 'A JavaScript error ocurred in the main process'

![image](https://user-images.githubusercontent.com/6452009/85375381-83a96680-b536-11ea-9111-70bea8ed36bf.png)

WALC Installation Type: AppImage
System Information (please complete the following information):
    OS/Distro: Fedora release 32 (Thirty Two)

**Comment by Zzombiee2361** [comment]
@erduende what's the output of
```
lsb_release -a
```

**Comment by MyMirelHub** [comment]
I get the same issue. 

```sh
➜  ~ lsb_release -a
No LSB modules are available.
Distributor ID:	Pop
Description:	Pop!_OS 20.04 LTS
Release:	20.04
Codename:	focal
```

These are the snap permissions: 

![Screenshot from 2020-06-23 10-27-55](https://user-images.githubusercontent.com/15373565/85387085-4cd64f00-b53c-11ea-947a-536d5b21e295.png)

**Comment by erduende** [comment]
@Zzombiee2361 
lsb_release output
```
LSB Version:	:core-4.1-amd64:core-4.1-noarch
Distributor ID:	Fedora
Description:	Fedora release 32 (Thirty Two)
Release:	32
Codename:	ThirtyTwo
```

**Comment by erduende** [comment]
The issue has been solved installing 'redhat-lsb-core' package to get the output of lsb_release. It seems that package isn't installed by default in Fedora.

Thank you!

**Comment by Zzombiee2361** [comment]
It seems this only happens on snap

**Comment by Gustav0ar** [comment]
I finally got the error to stop showing. I'm using PopOS 20.04, I installed the mainline kernel over the default one and it stopped showing.

**Comment by jvanstraten** [comment]
I also had the problem on OpenSUSE, again because lsb-release wasn't installed by default. I made a PR to fix/workaround, see #52.

**Comment by cstayyab** [comment]
PR Merged.
